### DISCUSSION TITLE
When configuring a Trino connection, if the URL scheme is http, consider updating the password column in the connection table to null.

### DISCUSSION THREAD

**Discussion by kyou-java** [apache/airflow#43772] [General]
Hello!

I would like to make a suggestion regarding Trino connection in Airflow UI.

When I first created the trino connection, I accidentally saved the password.
My Trino server only serves **http**.
Later, when I used the trino connection, I kept getting the error 'Authentication is not available over HTTP'.
As you know, trino does not support basic authentication over http.
However, since Airflow UI does not display the password for security reasons, I was able to see later that the password was still there in the PostgreSQL `connection` table. I had to manually update the value of the `password` column to **null** before it worked.

So, for the sake of the user, if the schema of the Trino connection URL is http, how about updating it to null when the password is not set?

Thanks for your feedback.

**Comment by potiuk** [comment]
Sure - if you want to make PR about it - feel free. Or open a feature issue describing it and we can mark it as "good first issue" for someone to pick up and work on.

**Reply by kyou-java** [reply]
Thank you for your help!
